api/src/main/java/pl/kolendateam/dadcard/feats/dto/FeatsDTO.java

```diff
@@ -12,14 +13,17 @@ public class FeatsDTO {
     public int id;
     public String featName;
     public FeatsTypeEnum featsType;
-    public String prerequisite;
+    public PrerequisiteDTO prerequisite;
     public String description;
 
     public FeatsDTO(Feats feats){
         this.id = feats.getId();
         this.featName = feats.getFeatName();
         this.featsType = feats.getFeatsType();
-        this.prerequisite = feats.getPrerequisite();
+        if(feats.getPrerequisite() == null){
```

Comment:
po co ten warunek? nie wystarczy this.prerequisite = feats.getPrerequisite();? wychodzi na jedno jak feats.getPrerequisite() to null to przypiszesz do this.prerequisite null

Reply:
trzeba dawać bo wywala że jest puste, i java się obraża
![image](https://github.com/Skalord88/ded_card/assets/123017102/b0f5bbe0-d1e5-4b79-97c7-c7725923d9f4)

Reply:
![image](https://github.com/Skalord88/ded_card/assets/123017102/5fc0dafc-d3b6-41ef-8b66-a187f01626fe)